apps/client/src/routes/dashboards/create/components/create-dashboard-form-actions/create-dashboard-form-actions.tsx

```diff
@@ -0,0 +1,21 @@
+import SpaceBetween from '@cloudscape-design/components/space-between';
+
+import { CancelCreateDashboardButton } from './cancel-create-dashboard-button';
+import {
+  ConfirmCreateDashboardButton,
+  ConfirmCreateDashboardButtonProps,
+} from './confirm-create-dashboard-button';
+
+type CreateDashboardFormActionsProps = ConfirmCreateDashboardButtonProps;
```

Comment:
Why reassign the type? If these are always going to be the same, I'd suggest updating the name to something still readable, but more generic (ex: `DashboardButtonProps`) so that you're not having to reassign duplicate values. It will also improve the readability in that someone going in will have a clear understanding that they are not unique more readily. They wouldn't know this if they went in on the "confirm" side of the codebase. 

If they do ever split off from each other then you'd be able to just extend the original type & maintain managing a single `type` value for what is shared between the two in honor of a DRY codebase.

Reply:
This will soon be converted to `CreateDashboardFormActionProps extends ConfirmCreateDashboardButtonProps` to prevent duplication of the type as props are added. At this time though, the linter will not allow me to do such. I would not think it hurts readability. The components are adjacent to each other. I would consider it less readable without the type alias. When using the component, my IDE would tell me the form action props are button props, which is not understandable without reading the implementation of the actions component.

Reply:
Sounds good. Thanks for the explanation.